azure/new-workspace/main.tf

```diff
@@ -262,6 +262,48 @@ SentFactsTotal: long)
     '    { "column" : "SentMessagesTotal", "Properties":{"Path":"$.sentMessagesTotal"}},'
     '    { "column" : "SentFactsTotal", "Properties":{"Path":"$.sentFactsTotal"}},'
     ']'
+//
```

Comment:
Yes + create the event hubs like lines 139->166 without the management of common event hub namespace, or you add in your API the code to handle event hub name $WORKSPACEID-scenariomedata and $WORKSPACEID-scenariorunmetdata event hub names (then no need to check on dedicated event hub flag)